lib/services/android/gradle-build-args-service.ts

```diff
@@ -0,0 +1,65 @@
+import * as path from "path";
+import { Configurations } from "../../common/constants";
+
+export class GradleBuildArgsService implements IGradleBuildArgsService {
+	constructor(private $androidToolsInfo: IAndroidToolsInfo,
+		private $logger: ILogger) { }
+
+	public getBuildTaskArgs(buildConfig: IAndroidBuildConfig): string[] {
+		const args = this.getBaseTaskArgs(buildConfig);
+		args.unshift(this.getBuildTaskName(buildConfig));
+
+		return args;
+	}
+
+	public getCleanTaskArgs(buildConfig: IAndroidBuildConfig): string[] {
+		const args = this.getBaseTaskArgs(buildConfig);
+		args.unshift("clean");
+
+		return args;
+	}
+
+	private getBaseTaskArgs(buildConfig: IAndroidBuildConfig): string[] {
+		const args = this.getBuildLoggingArgs();
+
+		const toolsInfo = this.$androidToolsInfo.getToolsInfo();
+		args.push(
```

Comment:
Are we omitting `-PtargetSdk` on purpose?

Reply:
We discussed the support for `targetSdk` version and as there are some unclear things around it we decided to do it in another PR.